http_client.rst

```diff
@@ -2032,6 +2032,19 @@ Then configure Symfony to use your callback:
             ;
         };
 
+You can also use :class:`Symfony\\Component\\HttpClient\\Response\\JsonMockResponse` which automatically encode
```

Comment:
We should explain before and after, like:

"To return json, you would normally do ......", CODE-EXAMPLE, "you can use the following way instead...." CODE-EXAMPLE with `JsonMockResponse`

WDYT?

Reply:
I push new version with your suggestion !

But I'm not sure about php code style, What is the rule ? Does pipeline check this ?